doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md

```diff
@@ -349,11 +344,10 @@ The feature has minimal impact on the memory consumption. Overall it requires le
 
 ### Unit Test cases ###
 
-1. Add new YANG model tests for the new MID_PLANE_BRIDGE and DPUS table.
-2. Extend existing YANG model tests to cover MGMT_INTERFACE table changes.
-3. Add new tests to cover the handling of MID_PLANE_BRIDGE and DPUS tables in sonic-cfggen utility.
-4. Add new tests to cover DHCP server configuration generation for the Smart Switch in sonic-cfggen utility.
-5. Extend interface-config.service tests to verify midplane bridge configuration generation.
+1. Extend existing YANG model tests to cover MGMT_INTERFACE table changes.
```

Comment:
Can you please review this test case also? There are no MGMT_INTERFACE changes required anymore.